index.html

```diff
@@ -2582,179 +2582,141 @@ <h2>CBOR</h2>
 machine-readable, language-independent data interchange format that is
-self-describing and has built-in semantics for interoperability.  With specific
-constraints, CBOR can support all JSON data types (including JSON-LD) for
+self-describing and has built-in semantics for interoperability.  CBOR prioritizes 
+the simplicity of the encoder and decoder over the size of the encoded data.
+With specific constraints, CBOR can support all JSON data types (including JSON-LD) for
 translation between the DID document model (described in <a
-href="#data-model">Data Model</a> and <a>DID Documents</a>) and other core
-representations.
+href="#data-model">Data Model</a> and the <a href="#core-properties"> Core Properties section</a> of the DID Document</a>).
       </p>
 
-      <p>
-Concise Data Definition Language (CDDL) [[RFC8610]] is a notation used to
-express Concise Binary Object Representation (CBOR), and by extension JSON Data
-Structures. The following notation expresses the DID Document model in CBOR
-representation with specific constraints for deterministic mappings between
-other core representations.
-      </p>
-
-      <pre class="example nohighlight" title="DID Document data model for CBOR expressed in CDDL notation">
-DID-document = {
-  ? @context : uri
-  id : did
-  ? publicKey : [* publicKey ]
-  ? authentication :  [ *did // *publicKey // *tstr ]
-  ? service : [ + service  ]
-  ? controller : did / [ *did ]
-  ? created : time
-  ? updated : time
-  proof :  any
-}
-
-publicKey = {
-  id : did
-  type : text
-  controller : uri
-}
-
-did = tstr .pcre "^did\\:(?&lt;method-name>[a-z0-9]{2,})\\:(?&lt;method-specific-id>[A-Za-z0-9\\.\\-\\:\\_]+)"
-
-did-url =  tstr .pcre "^did\\:(?&lt;method-name>[a-z0-9]{2,})\\:(?&lt;method-specific-id>[A-Za-z0-9\\.\\-\\:\\_]+)\\;(?&lt;path>[A-Za-z0-9\\/)(?&lt;query>\\?[a-z0-9\\=\\&amp;])#(?&lt;fragment>.+)"
-
-service = {
-  id : did-url
-  type : text
-  serviceEndpoint : uri
-  ? description : text
-  * tstr => any
-}
-      </pre>
-
+<p>
+When producing and consuming DID documents that are encoded in CBOR (as indicated 
+by a content-type of application/did+cbor in the resolver metadata), the following rules MUST be followed.
+</p>
+      
       <section>
         <h3>Production</h3>
         <p>
-When producing DID Documents that are represented as CBOR, in addition to the
-suggestions in section 3.9 of the CBOR [[RFC7049]] specification for
-deterministic mappings, the following constraints of the DID Document model
-MUST be followed:
-        </p>
+A DID document encoded in CBOR must be represented as a CBOR Map (major type 5) conforming to [[RFC7049]]. 
+Similar to the JSON encoding of a DID document, all top-level properties of the DID document MUST be represented by using the 
+property name as the name of the member of the CBOR map.  The values of properties of the data model described in <a href="#core-properties"> § 5. Core Properties</a>, including 
```

Comment:
We should point to the CBOR spec for all type definitions.

Reply:
will do

Reply:
Done, but it just highlighted all of the problems of the ADM including the lack of clarity on number representation that we discussed and the gap with INFRA.  I added a bunch more details regarding bigNum and BigFloat and yet we don't need those types in the specification.  If we want to dwell on these, then I can make `lossless` conversions between CBOR and JSON via an ADM (ideally in CDDL), but it may be a exercise in futility.